lib/validator.js

```diff
@@ -18,128 +18,64 @@ const oneOf = (buffer, ...fns) => {
     return false;
 };
 
-const isPng = (buffer) => {
+/**
+ * Generic buffer check
+ * @param buffer {Buffer}
+ * @returns {boolean|Error}
+ */
+const checkBuffer = (buffer) => {
     if (!Buffer.isBuffer(buffer)) {
         throw new Error('second param should be buffer');
     }
 
     if (buffer.length <= 0) {
         return false;
     }
+}
 
-    if (Buffer.compare(s.PNG, buffer.subarray(0, s.PNG.length)) != 0) {
-        return false;
-    }
+/**
+ * Generic compare buffer fn wrapper
+ * @param buffer {Buffer}
+ * @param bufferType {Buffer}
+ * @returns {boolean|Error}
+ */
+const genericCompareBuffer = (buffer, bufferType) => {
+    checkBuffer(buffer);
+    return Buffer.compare(bufferType, buffer.subarray(0, bufferType.length)) == 0;
+}
 
-    return true;
+const isPng = (buffer) => {
+    genericCompareBuffer(buffer, s.PNG);
```

Comment:
> @lutfailham96 , mas i think the `return` keyword is mandatory ya mas, for all function. Since the `genericCompareBuffer` also returning a bool value

oh ya forgotten mas, already fixed